templates/bundles/DemosPlanCoreBundle/DemosPlanProcedure/administration_convert_annotated_pdf.html.twig

```diff
@@ -1,7 +1,14 @@
 {% extends '@DemosPlanCore/DemosPlanCore/procedure.html.twig' %}
 
 {% block component_part %}
-    {% set allTags = templateVars.availableTopics|default([])|map(el => { id: el.id, title: el.title, tags: el.tags|map(tag => { id: tag.id, name: tag.title }) }) %}
+    {% set allTags = templateVars.availableTopics|default([])|map(el => {
+        id: el.id,
+        title: el.title,
+        tags: el.tags|map(tag => {
+            id: tag.id,
+            title: tag.title
```

Comment:
the only real change I did in this block is to rename `name: tag.title` to `title: tag.title` the rest ist just reformating